bolt/test/X86/Inputs/cfg_build_hlt.s

```diff
@@ -0,0 +1,11 @@
+  .global main
+  .type main, %function
+main:
+        pushq   %rbp
+        movq    %rsp, %rbp
+        hlt
+        movl    $0x0, %eax
+        popq    %rbp
```

Comment:
Since we are not interested in other instructions, just leave `hlt` followed by `retq`.